README.md

```diff
@@ -22,7 +20,11 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Configure slack_logger with a file inside of your config/initializers directory.
```

Comment:
A code sample here would be very helpful

Reply:
Could you please use the code markup syntax so the code sample is nicely formatted?